install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -7,8 +7,6 @@
     You have preserved the configMap and successfully installed {{ title .Chart.Name}}.
 {{- else if (and (.Values.global.hubble.enabled) (.Values.global.hubble.ui.enabled)) }}
     You have successfully installed Cilium with hubble-ui.
-{{- else if (and (.Values.global.hubble.enabled) (.Values.global.hubble.cli.enabled) (.Values.global.hubble.ui.enabled)) }}
-    You have successfully installed Cilium with hubble-ui and hubble-cli.
```

Comment:
Is this change intentional? Most of the PR is about Hubble relay but this change isn't.

Reply:
no this is not intentional, let me clean this up